src/main/scala/com/github/music/of/the/ainur/almaren/mongodb/MongoDbConnector.scala

```diff
@@ -14,14 +14,17 @@ private[almaren] case class SourceMongoDb(
   collection: String,
   user:Option[String],
   password:Option[String],
+  stringPrefix:Option[String],
```

Comment:
Update the documentation and explain what's this option.

Reply:
Please check the Documentation updated parameter description.